### ISSUE TITLE
SeriesGroupBy.sum() not recognising `min_count` parameter

### ISSUE THREAD

**Issue by atlasstrategic** [dask/dask#4586]
#### Code Sample

```python
import numpy as np
import pandas as pd
import dask as dk

print(dk.__version__)
# 1.1.4

print(pd.__version__)
# 0.23.4

df = pd.DataFrame({
    'code':['A','A'],
    'val1':[1,2],
    'val2':[np.nan,np.nan],
    'val3':[4,np.nan],
    'val4':[5,0]
})
df = dk.dataframe.from_pandas(df, npartitions=1)

df = df.groupby(['code'])['val1','val2','val3','val4'].sum(min_count=1).reset_index()

# TypeError: sum() got an unexpected keyword argument 'min_count'

```
#### Problem description

As shown for `DataFrame` in https://github.com/dask/dask/issues/3981 and fixed with https://github.com/dask/dask/pull/4090 there is still support needed for `dask.dataframe.groupby.SeriesGroupBy`?

#### Expected Output

`df.head()`

<table border="1" class="dataframe"><thead> <tr style="text-align: right;"><th></th><th>code</th><th>val1</th><th>val2</th><th>val3</th> <th>val4</th></tr></thead><tbody> <tr><th>0</th><td>A</td><td>3</td><td>NaN</td><td>4.0</td><td>5</td></tr></tbody></table>

**Comment by TomAugspurger** [comment]
Likely just not implemented yet. A fix would look somewhat similar to #4090.

**Comment by jrbourbeau** [comment]
Closing as `min_count` was added in #4648. Thank you for the bug report @atlasstrategic.
